test/user.test.js

```diff
@@ -1804,6 +1805,360 @@ describe('User', function() {
     });
   });
 
+  describe('Email Update', function() {
+    describe('User changing email property', function() {
+      var user;
+      var currentEmailCredentials = { email: 'original@example.com', password: 'bar' };
+      var updatedEmailCredentials = { email: 'updated@example.com', password: 'bar' };
+
+      beforeEach('create user then login', function createAndLogin(done) {
+        async.series([
+          function createUserWithOriginalEmail(next) {
+            User.create(
+              { email: currentEmailCredentials.email,
+                password: currentEmailCredentials.password },
+                function(err, userCreated) {
+                  if (err) return next(err);
+                  user = userCreated;
+                  next();
+                });
+          },
+          function firstLoginWithOriginalEmail(next) {
+            User.login(
+              { email: currentEmailCredentials.email,
+                password: currentEmailCredentials.password },
+                function(err, accessToken1) {
+                  if (err) return next(err);
+                  assert(accessToken1.userId);
+                  next();
+                });
+          },
+          function secondLoginWithOriginalEmail(next) {
+            User.login({
+              email: currentEmailCredentials.email,
+              password: currentEmailCredentials.password },
+              function(err, accessToken2) {
+                if (err) return next(err);
+                assert(accessToken2.userId);
+                next();
+              });
+          },
+        ], function(err) {
+          done();
+        });
+      });
+
+      afterEach('Deleting original user', function deleteOriginalAccounts(done) {
+        User.deleteById(user.id, function(err, user) {
+          if (err) return done (err);
+          done();
+        });
+      });
+
+      it('invalidates sessions when email is changed using `updateAttributes`', function(done) {
+        user.updateAttributes({ 'email': updatedEmailCredentials.email },
+        function(err, userInstance) {
+          if (err) return done(err);
+          assertNoAccessTokens(done);
+        });
+      });
+
+      it('invalidates sessions when email is changed using `replaceAttributes`', function(done) {
+        user.replaceAttributes(
+          { 'email': updatedEmailCredentials.email,
+            'password': updatedEmailCredentials.password },
```

Comment:
coding style: one arg per line. see https://github.com/strongloop/loopback-contributor-docs/blob/master/style-guide.md#one-arg-per-line

This particular place is not even consistent with the style you are using above.